util/config/run-in-venv-with-python-bindings.bash

```diff
@@ -17,4 +17,8 @@ if [ ! -d "$chpl_frontend_py_deps/chapel" ]; then
   exit 1
 fi
 
+if [ "$CHPL_SANITIZE" == "address" ]; then
```

Comment:
You should not rely on the env var `CHPL_SANITIZE`, instead use `$("$python" "$CHPL_HOME/util/chplenv/chpl_sanitize.py")`